addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SstiScanRule.java

```diff
@@ -71,23 +73,27 @@ public class SstiScanRule extends AbstractAppParamPlugin implements CommonActive
     // the first and last tag are the commentaries from twig
     // "\\\"\\<th\\:t\\=\\$\\{zj\\}\\#\\foreach"
 
-    private static final TemplateFormat[] TEMPLATE_FORMATS = {
-        new TemplateFormat(" ", " "),
-        new TemplateFormat("{", "}"),
-        new TemplateFormat("${", "}"),
-        new TemplateFormat("#{", "}"),
-        new TemplateFormat("{#", "}"),
-        new TemplateFormat("{@", "}"),
-        new TemplateFormat("{{", "}}"),
-        new TemplateFormat("{{=", "}}"),
-        new TemplateFormat("<%=", "%>"),
-        new TemplateFormat("#set($x=", ")${x}"),
-        new TemplateFormat("<p th:text=\"${", "}\"></p>"),
-        new TemplateFormat(
-                "{", "}", "{@math key=\"%d\" method=\"multiply\" operand=\"%d\"/}"), // Dustjs
-        new DjangoTemplateFormat(),
-        new GoTemplateFormat()
-    };
+    static List<TemplateFormat> DEFAULT_TEMPLATE_LIST =
+            List.of(
+                    new TemplateFormat(" ", " "),
+                    new TemplateFormat("{", "}"),
+                    new TemplateFormat("${", "}"),
+                    new TemplateFormat("#{", "}"),
+                    new TemplateFormat("{#", "}"),
+                    new TemplateFormat("{@", "}"),
+                    new TemplateFormat("{{", "}}"),
+                    new TemplateFormat("{{=", "}}"),
+                    new TemplateFormat("<%=", "%>"),
+                    new TemplateFormat("#set($x=", ")${x}"),
+                    new TemplateFormat("<p th:text=\"${", "}\"></p>"),
+                    new TemplateFormat(
+                            "{",
+                            "}",
+                            "{@math key=\"%d\" method=\"multiply\" operand=\"%d\"/}"), // Dustjs
+                    new DjangoTemplateFormat(),
+                    new GoTemplateFormat());
+
+    private static List<TemplateFormat> templateFormats = DEFAULT_TEMPLATE_LIST;
```

Comment:
Preferable for this to be an instance variable, which does not require clean ups. (Though these whole changes would not be needed the Go expression is very specific and could be matched in the server.)